freeipa/src/main/java/com/sequenceiq/freeipa/converter/backup/BackupConverter.java

```diff
@@ -39,6 +40,18 @@ public Backup convert(TelemetryRequest request) {
         return backup;
     }
 
+    public Backup convert(BackupRequest request) {
+        Backup backup = null;
+        if (freeIpaBackupEnabled && request != null) {
```

Comment:
when `freeIpaBackupEnabled` is `false` or `request` is `null` is not tested

Reply:
Thanks! Now I added test cases for freeIpaBackupEnabled is false or request is null.

This is a copy of the original working code 
`
    public Backup convert(TelemetryRequest request) {
        Backup backup = null;
        if (freeIpaBackupEnabled && request != null && request.getLogging() != null) {
`

Reply:
Thanks! Now I added test cases for freeIpaBackupEnabled is false or request is null.

This is a copy of the original working code

```
    public Backup convert(TelemetryRequest request) {
        Backup backup = null;
        if (freeIpaBackupEnabled && request != null && request.getLogging() != null) {
```